libs/facilities/frontend/view/src/lib/directives/back-button.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Location } from '@angular/common';
+import { Directive, HostListener } from '@angular/core';
+
+@Directive({
+    standalone: true,
+    // eslint-disable-next-line @angular-eslint/directive-selector
+    selector: '[backButton]',
+})
+export class BackButtonDirective {
+    constructor(private location: Location) { }
+
+    @HostListener('click')
+    onClick() {
+        this.location.back();
+    }
```

Comment:
Passt super! Kannst du vlt noch die Sektor rule in der eslint rule (im root Ordner) deaktivieren?

Reply:
Die rule steht in der .eslintrc.json vom Root Ordner drin und in der .eslintrc.json von den libs. Es nur im root Ordner zu deaktivieren reicht nicht. Muss ich das dann in den einzelnen rauslöschen?